### PR TITLE
feat: Added support for django3.2

### PR DESCRIPTION
**Pull request by Jawayria**
JIRA: https://openedx.atlassian.net/browse/BOM-2619

### CODE REVIEW

File: .github/workflows/upgrade-python-requirements.yml [hunk 1]
```diff
@@ -1,66 +1,69 @@
 name: Upgrade Requirements
 
- on:
-   schedule:
-     # will start the job at 07:00 UTC every Wednesday
-     - cron: "0 7 * * 3"
-   workflow_dispatch:
```
Comment (mraarif) [inline .github/workflows/upgrade-python-requirements.yml:7]: we should undo these changes (I'm assuming) made the modernized script

File: .github/workflows/upgrade-python-requirements.yml [hunk 2]
```diff
@@ -1,66 +1,69 @@
...
-         run: |
-           git clone https://github.com/edx/testeng-ci.git
-           cd $GITHUB_WORKSPACE/testeng-ci
-           pip install -r requirements/base.txt
-       - name: create pull request
-         env:
-           GITHUB_TOKEN: ${{ secrets.REQUIREMENTS_BOT_GITHUB_TOKEN }}
-           GITHUB_USER_EMAIL: ${{ secrets.REQUIREMENTS_BOT_GITHUB_EMAIL }}
-         run: |  
-           cd $GITHUB_WORKSPACE/testeng-ci
```
Comment (mraarif) [inline .github/workflows/upgrade-python-requirements.yml:49]: I don't see a need for this change

File: tox.ini [hunk 3]
```diff
@@ -13,61 +13,62 @@ ignore = D101,D200,D203,D204,D205,D212,D215,D400,D401,D404,D405,D406,D407,D408,D
...
+    python -Wd -m pytest {posargs}
 
 [testenv:docs]
 setenv = 
-	DJANGO_SETTINGS_MODULE = test_settings
-	PYTHONPATH = {toxinidir}
+    DJANGO_SETTINGS_MODULE = test_settings
+    PYTHONPATH = {toxinidir}
 whitelist_externals = 
-	make
```
Comment (mraarif) [inline tox.ini:37]: same here

**Review by mraarif — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Review by awais786 — APPROVED** [review]

**Comment by Jawayria** [comment]
Warnings not fixed under the scope of this PR
```
RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
    help_text=_(

RemovedInDjango40Warning: django.utils.translation.ugettext() is deprecated in favor of django.utils.translation.gettext().
    label=_('User Email'),
```
